### DISCUSSION TITLE
HOW TO FIND THE PW TO 500/350/M1/M2

### DISCUSSION THREAD

**Discussion by socelillyhammer** [GerryDazoo/Slinger#374] [Q&A]
Ok so for all who did not log in to slingbox in to get your password you can still get your password!!!!

Steps:
1. download slingplayer desktop version 5.0.0.83
2. install it
3. log in with your standard username and password
4. connect
5. it will try and deny you access becuase sb shut down
6. then go into your directory C:\Users\________\AppData\Roaming\Sling Media\Slingplayer Desktop\Logs
7. go to the console file
8. the password and your ID are in there!!!!

@GerryDazoo  i need help now getting the video to come up and then figure out how to access all of this remotely
@Toei79 @rbf121 @Dang638

I hope this helps out everyone to find their PW :-)

Please help all me now with the next steps

**Comment by GerryDazoo** [comment]
If this works then you deserve all the help you need!
Where are you in the process?

**Reply by socelillyhammer** [reply]
I can see the video and audio on my comp and on my android with VLC.  Though it takes some time to connect to VLC as I have to keep doing it over and over.  I saw a discussion about that.  Need time to read and understand it all.

Now I am trying to figure out how to get the FIOS remote to appear on VLC windows 10 and android.

Then once I have that going, I'll need to figure out how to add my second sling box to the server config file, then figure out how to deal with a remote login.

I read your readme pdf and release notes.  For all your hard work yes we should all learn how to redirect the port properly. If you need to adjust the config, pls do so.

Ive been up for hours learning how to read all of the coding and still dont understand.  current config file is attached.  Time is money so please, i'll pay a service fee to help me get up and running and an annual fee of $300

[SLINGBOX.docx](https://github.com/GerryDazoo/Slinger/files/10111930/SLINGBOX.docx)

**Reply by GerryDazoo** [reply]
You can call up the remote on any old browser. In your config.ini you need to tell the code which remote file to use. Change
;include=remote.txt
to
include=CustomRemotes/FIOS_remote.txt

The you can connect to  http://ip:8080/Remote

BTW. You are not using the unified config file format. Which I would recommend when your ready to add your second box. Let us know when you've got this working and we'll move on to your next step.

**Reply by Toei79** [reply]
i see. i did exactly that thing in mine.  i install app and i found console.log in my actual pc.  i able to log in and see slingboxes but not more.

i have old pc but never found console.log
i never though it was some important advance, probably because i dont get it works , i have 3 units one pro two 350 
the only one working its the one that i reset password.

**Comment by DoogieHowitzerWMD** [comment]
Thanks for sharing what you've found. That's awesome news if that works as described.

Assuming it does, this already has me thinking about some of the implications:

1. Would this only work with 500's that have already been configured (one's that already had an email and user-defined password)? If so, this would still help those who were unable to get their admin password before the server shutdown.
2. Would that console file still manage to be generated for a slingbox that has since been blocked from outside communications? I suppose that would be a question of where the info filling the console file is being pulled from.

It will be interesting to see what develops from this.

**Reply by socelillyhammer** [reply]
To answer your question
1. I have to assume so becuase when i logged in both of my slingboxes were still showing up in my account
2. this I dont know as I have the 500s but both of my boxes were blocked from accessing slingbox.com

**Comment by Toei79** [comment]
l already did that.  i extract all info from my console.log file

then one slingbox that i did reset works fine. but those not

**Reply by socelillyhammer** [reply]
If your boxes are showing up in the desktop interface it should work.  If they are removed from the slinbox desktop interface I think you are out of luck

**Comment by rbf121** [comment]
Nice find!  I knew the account box passwords appear in the console logs if you ever used Slingplayer Desktop but I had no idea that folks could still log in after Nov 9 (I never tried).  I guess that means that some of the APIs are still running and they only took down the main website.

**Comment by carranzafp** [comment]
It also works with Slingbox 350, I can see on the logs the hashed passwords of the 2 units that I have
Make sure you see the directory and dont just quit with the first error screen (click retry)

**Reply by socelillyhammer** [reply]
I'm glad to hear that it worked for you.

**Reply by regmar505** [reply]
I have a Slingbox M2 and I know the admin password. Out of curiosity, I installed Slingbox Desktop on a VirtualBox Windows guest. I entered my Sling login info and as a matter of fact, console.log was created and I found the E1: password. I then replaced my "short" password with E1: and Slinger decrypted it like a charm. Nice finding @socelillyhammer , it should help a lot of users.

**Reply by socelillyhammer** [reply]
YW glad to help out

**Comment by RandomMan719** [comment]


**Reply by Toei79** [reply]
You need just the password  or also the log in , nick name in the .ini file?

**Reply by Toei79** [reply]
I only use in my hone understanding  that you say

I can't advance so far because  , I don't able get it work 

I will do a video
Thanks for all the help

**Comment by womeara6** [comment]
Hi.  I followed the steps and when I execute the Slingplayer desktop the slingbox 500 shows up with the green light lit. I hit the connect button and waited for it to fail. I looked at the consol.log file but can't find the password.  How is it described in the file or where can I look?

Thanks in advance.

**Reply by Toei79** [reply]
you need looks for E1: , or check long messages usually its there.  log file its long text at least in mine.

**Comment by womeara6** [comment]
Thank you Toei79.  I found the password for my 500 but there was also a password for a Solo box that has been off line for quite a while.  There was also mention of an ID.  What does the ID refer to?  the slingbox?

Thanks again!!

**Reply by Toei79** [reply]
> Thank you Toei79. I found the password for my 500 but there was also a password for a Solo box that has been off line for quite a while. There was also mention of an ID. What does the ID refer to? the slingbox?
> 
> Thanks again!!

you need check finderid , when you run script in commmand line it detects your finderid, then you need see which one match. 

or easier try both passwords. 

in my case i have 3 slingboxes. i did in that way , with finderid.  where its your password in the console.log its also the finderid.

**Comment by robosmith** [comment]
Got a used 500 off ebay and installed the SPDesktop 5.083. Running with no Internet I get a black screen with fail message.
No sign of a console.log in the C:\Users\me\AppData\Roaming\Sling Media folder

**Reply by rbf121** [reply]
This thread is about recovering device passwords for boxes that were already configured before the Nov 9 shutdown, assuming you still know your sling login info.  If you just got the device then I think your only option would be to factory reset and follow what this guy did: https://www.reddit.com/r/slingbox/comments/yvkowq/i_figured_out_a_way_to_use_a_reset_slingbox_on/

Or there was someone on reddit who was talking about a typescript app he wrote to configure slingbox 500, not sure if what he posted on github is workable but maybe worth checking out:
https://github.com/Nomkid/Blingsox

**Reply by socelillyhammer** [reply]
Thank you for the link.  I was looking for those two again.
Yes. robosmith, if you dont have the user name login and the password that was previously used for that slingbox when you wanted to sign in and watch it,  you are not getting that PW from this method.

**Reply by GerryDazoo** [reply]
rb121. The 5.x must have internet to work. It's basically an empty shell and downloads JavaScript from the SB servers to do what it needs to do.

**Reply by GerryDazoo** [reply]
RE: blingsox. It is just a player. Which we don't need. I was trying to convince him to write the reconfigure  code but it didn't happen.

**Comment by garpinc** [comment]
This worked great for me. I had an old PC where I had used sling desktop and the log was still there in C:\Users________\AppData\Roaming\Sling Media\Slingplayer Desktop\Logs

I missed the slingmedia shutdown so were it not for finding this file I would not have been able to salvage this slingbox
